templates/terraform/constants/deployment_manager_deployment.go.erb

```diff
@@ -0,0 +1,21 @@
+func customDiffDeploymentManagerDeployment(d *schema.ResourceDiff, meta interface{}) error {
+	preview := d.Get("preview").(bool)
```

Comment:
nit: I like this as an `if` statement slightly more, I missed the guard on my first glance at this and thought L10 happened on any change.